packages/user-banning-nodejs/src/plugin.ts

```diff
@@ -0,0 +1,197 @@
+import { SuperTokensPlugin } from 'supertokens-node/types';
+import SuperTokens from 'supertokens-node';
+import Session from 'supertokens-node/recipe/session';
+import { PLUGIN_ID, HANDLE_BASE_PATH, PLUGIN_SDK_VERSION } from './config';
+import { UserBanningService } from './userBanningService';
+import { PermissionClaim } from 'supertokens-node/recipe/userroles';
+
+// todo: feedback: have more exposed apis from the sdk - can't list users, cant get a user by email (or other fields), etc.
+
+export const init = (): SuperTokensPlugin => {
+  const userBanningService = new UserBanningService();
+
+  return {
+    id: PLUGIN_ID,
+    compatibleSDKVersions: PLUGIN_SDK_VERSION,
+    routeHandlers: [
```

Comment:
It'd be nice if we had a way to handle this from the dashboard. Maybe if we used roles instead of user metadata? If we supported both doing it through the dashboard and the FE plugin/component, then this would make sense separately as well.